proposals/inline-arrays.md

```diff
@@ -113,17 +113,20 @@ For an inline array element access, the *primary_no_array_creation_expression* o
 
 If *primary_no_array_creation_expression* is a writable variable, the result of evaluating an inline array element access is a writable variable
 equivalent to invoking [`public ref T this[int index] { get; }`](https://learn.microsoft.com/en-us/dotnet/api/system.span-1.item?view=net-8.0) with
-that integer value on an instance of ```System.Span<T>``` returned by ```System.Span<T> InlineArrayAsSpan``` method on *primary_no_array_creation_expression*. 
+that integer value on an instance of ```System.Span<T>``` returned by ```System.Span<T> InlineArrayAsSpan``` method on *primary_no_array_creation_expression*. For the purpose of ref-safety analysis the *ref-safe-to-escape*/*safe-to-escape*
+of the access are equivalent to the same for an invocation of a method with the signature ```static ref T GetItem(ref InlineArrayType array)```.
```

Comment:
Was the original problem that the `this` parameter is implicitly `scoped ref`, but the element access is supposed to return a ref to the element within `this`? And I guess a similar issue in the other changed sections?

Reply:
Correct.